homeassistant/components/velux/entity.py

```diff
@@ -19,7 +23,18 @@ def __init__(self, node: Node, config_entry_id: str) -> None:
             if node.serial_number
             else f"{config_entry_id}_{node.node_id}"
         )
-        self._attr_name = node.name if node.name else f"#{node.node_id}"
+        self._attr_device_info = DeviceInfo(
+            identifiers={
+                (
+                    DOMAIN,
+                    self.node.serial_number
+                    if self.node.serial_number
+                    else str(self.node.node_id),
+                )
+            },
```

Comment:
Why do we set the unique id to config entry id + node id and the device to just node id?

I would opt to make them the same (and ideally we don't use the config entry id, but that depends on what the node id is)

Reply:
Probably we should try to combine with config_entry unique_id first (istead of entry_id) if unique_id is available? Then the user-data (i18n of entity names) may probably remain, if the same KLF200 will be setup again? I'm not sure when storage is cleaned up for config_entries within homeassistant. @joostlek can you advise?